### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Elanis**

### CODE REVIEW

**Comment by Equinox4** [comment]
Could we use a yaml file instead of markdown for "proposer-un-article"?
Yaml allows you to configure some things more finely, like adding a required field for example.

**Comment by Elanis** [comment]
> Could we use a yaml file instead of markdown for "proposer-un-article"? Yaml allows you to configure some things more finely, like adding a required field for example.

Dunno how issues templates work tbh, used the default given thing.
I guess you're talking about [Issues forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) ?

I can convert existing templates in this PR to yaml I guess

**Comment by Equinox4** [comment]
Yes, I think you can just rename .md to .yml but I don't know yet how to preview the templates to check the rendering

**Comment by Elanis** [comment]
I did switch to yaml, creating forms with https://issue-forms-creator.app/new

File: .github/ISSUE_TEMPLATE/proposer-un-article.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Proposer un article
+description: Proposer un nouvel article pour le blog
+title: '[NOUVEL ARTICLE] '
+body:
+  - type: textarea
+    id: sujet
```
Comment (Equinox4) [inline .github/ISSUE_TEMPLATE/proposer-un-article.yaml]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/proposer-un-article.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Proposer un article
+description: Proposer un nouvel article pour le blog
+title: '[NOUVEL ARTICLE] '
```
Comment (Equinox4) [inline .github/ISSUE_TEMPLATE/proposer-un-article.yaml:3]:
```suggestion
title: '[NOUVEL ARTICLE] '
labels: ["article"]
```

**Review by Equinox4 — CHANGES_REQUESTED** [review]
id is optionnal + automatically add labels on issue

**Review by Equinox4 — APPROVED** [review]

**Comment by Elanis** [comment]
Thoughts @Julien00859 @arkena00 ?

**Comment by Elanis** [comment]
If you feel there need another template, feel free to open an issue or a pull request

**Pull request merged**